identity/manager.go

```diff
@@ -0,0 +1,93 @@
+package identity
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"fmt"
+	"github.com/mysterium/node/service_discovery/dto"
+	"strings"
+)
+
+const PASSPHRASE = ""
```

Comment:
needs a TODO or FIXME tag comment here so that constant empty password doesn't reach production

Reply:
Passphrase now can be configurable argument for factory, as `keydir`